cmd/system-probe/modules/oom_kill_probe.go

```diff
@@ -29,7 +29,7 @@ var OOMKillProbe = module.Factory{
 	ConfigNamespaces: []string{},
 	Fn: func(cfg *config.Config) (module.Module, error) {
 		log.Infof("Starting the OOM Kill probe")
-		okp, err := probe.NewOOMKillProbe(ebpf.NewConfig())
+		okp, err := oomkill.NewOOMKillProbe(ebpf.NewConfig())
```

Comment:
As the name of the check (`oomkill`) is now part of the package name, would it make sense to simplify the name of the function ?

```suggestion
		okp, err := oomkill.NewProbe(ebpf.NewConfig())
```